spanner/benchmarks/benchmarks.go

```diff
@@ -229,6 +248,32 @@ func executeRead(ctx context.Context, client *spanner.Client, staleness int64) (
 	return time.Since(start), nil
 }
 
+func enableTracingWithCloudTraceExporter(projectID string, cloudTracingHost string) {
+	res, err := resource.Merge(resource.Default(),
+		resource.NewWithAttributes(semconv.SchemaURL,
+			semconv.ServiceName("Go-Benchmarking"),
+			semconv.ServiceVersion("0.1.0"),
+		))
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Create a new cloud trace exporter
+	exporter, err := traceExporter.New(traceExporter.WithProjectID(projectID), traceExporter.WithTraceClientOptions([]option.ClientOption{option.WithEndpoint(cloudTracingHost)}))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	tracerProvider := sdktrace.NewTracerProvider(
+		sdktrace.WithResource(res),
+		sdktrace.WithBatcher(exporter),
+		sdktrace.WithSampler(sdktrace.TraceIDRatioBased(0.1)),
```

Comment:
10% is a relatively high trace ratio. You might want to make this configurable and/or reduce the default value